packages/utilities/psammead-styles/src/font-styles.js

```diff
@@ -0,0 +1,38 @@
+import * as fonts from './font-families';
+
+export const getSansRegular = service =>
+  fonts[service] && fonts[service].sansRegular;
+
+export const getSansItalic = service => {
+  if (!fonts[service]) {
+    return undefined;
+  }
+  const { sansItalic, sansRegular } = fonts[service];
+  return sansItalic || sansRegular;
+};
+
+export const getSansBold = service => fonts[service] && fonts[service].sansBold;
```

Comment:
this fails if sans bold isnt defined, should it fall back to regular?

Reply:
yeah I believe it should.